### ISSUE TITLE
CSV with embeded geojson examples

### ISSUE THREAD

**Issue by PybloBenavides** [keplergl/kepler.gl#736]
Hello everyone, 

Thanks for this amazing open source project. 
I am trying to upload a csv with ODs and LineStrings in between them so I can display all of them together. The user doc sais that "Kepler.gl attempts to parse geometry from your CSV file if the column content contains geometry data types.". The problem I have is that kepler is not parsing ok the column where I have my geojson object. This is due to the fact that the coordinates of the geojson object are also separated by commas. 
I have tried separating the csv with semicolons or tabs but it would not do the trick, only commas seem to be supported. The geojson I have tried to code with LineStrings and with Features, same result, also tried putting them between ""
[test-5Lines.txt](https://github.com/keplergl/kepler.gl/files/3616040/test-5Lines.txt) (Github only allows me to upload txt not csv but it's meant to be a csv)

.

If this is my mistake, which it probably is, I haven't found an example on how to code this for the upload. 

Thank you very much in advance!

**Comment by vanhumbeecka** [comment]
Same here, 
It used to work in the past though (somewhere short after version 1.0.0 i think, did it work)
Now, with my same dataset, the CSV upload seems broken for `geojson` entries.
I'm also trying to upload LineString geojson.

**Comment by heshan0131** [comment]
Hi @vanhumbeecka 
I thinks your csv file is missing the string escape. I added quotes to object keys `"type"` and `"coordinates"`, it loads fine. 

Need to add quote to escape `"`. This works for me:
```
trip.geojson,trip_id,pickup_lat,pickup_lng,pickup_datetime,dropoff_lat,dropoff_lng,dropoff_datetime
"{""type"": ""LineString"",""coordinates"": [[-58.3867141, -34.6016724], [-58.388046, -34.601724], [-58.3895469, -34.6018105], [-58.3894607, -34.6030247], [-58.3894148, -34.6041908], [-58.3894125, -34.6042488], [-58.3893415, -34.6054426], [-58.3893196, -34.6066263], [-58.3892645, -34.6078317], [-58.3891868, -34.609105], [-58.389166, -34.6092809], [-58.3890649, -34.610095], [-58.3889687, -34.6112764], [-58.3889168, -34.6124787], [-58.3903475, -34.6125322], [-58.3903555, -34.6137279], [-58.3903758, -34.614812], [-58.3903721, -34.6156194]]}",5453,-34.6018635,-58.3866934,2018-01-01 00:08:05,-34.6156994,-58.3899728,2018-01-01 00:27:58
```
<img width="1097" alt="Screen Shot 2019-11-09 at 2 29 51 AM" src="https://user-images.githubusercontent.com/3605556/68527167-d7618b00-0298-11ea-8948-c5fa05250093.png">

Here is more info on [string escape in csv](https://gpdb.docs.pivotal.io/43250/admin_guide/load/topics/g-escaping-in-csv-formatted-files.html)

Basically when your json is formatted as a string in a single cell:
 `{"type": "LineString", "coordinates": [1,2]}`, 
you have to use `"` to escape the `"`, and wrap the entire string in `"` so it becomes:

`"{""type"": ""LineString"", ""coordinates"": [1,2]}"`

I will add note to this in our documentation

**Comment by vanhumbeecka** [comment]
I've added a CSV snippet in #818 . Based on the the geojson-structure you mention, this seems to be fine. So I'm still not sure where the problem could be.
